.travis.yml

```diff
@@ -36,32 +80,37 @@ jobs:
             docker push ${DOCKER_REPO}:${DOCKER_NTAG}
             docker push ${DOCKER_REPO}:latest
           fi
-    - stage: build
+
+    - name: Build FreeOrionSDK for MacOS
+      <<: *_build_sdk_common
       os: osx
-      language: cpp
-      osx_image: xcode8.3
+      osx_image: xcode9.4
       compiler: clang
-      cache:
-        directories:
-          - build/downloads
       env:
-        - NAME="Build FreeOrionSDK"
+        - RELEASES_FILE=FreeOrionSDK_*.tar.bz2
+        - RELEASES_TAG_NAME=$TRAVIS_TAG
       before_install:
         - brew install gpatch
-      before_script:
-        - mkdir -p build
-        - cd build
       script:
         - cmake -G "Xcode" -D BOOST_PARALLEL_JOBS=$(sysctl hw.ncpu | awk '{print $2}') ..
         - travis_wait 70 cmake --build . --config RelWithDebInfo -- -parallelizeTargets -jobs $(sysctl hw.ncpu | awk '{print $2}')
-      deploy:
-        provider: releases
-        api-key:
-          secure: "fRPiVwWwyhGlR+F+Vzt9ZDszc+TXrkq+UZ8uYRT//H0NrYV+OvvkAhE4GxRqDC2SfTnwJ5wCc07yf2Prxv7Zm5KYAJcCjBA04L9D/MD03+McGJIiKpg86XEOwAAUGuCB74gyCj3tuIKkQsops8HzYyNsP10q0o0kXQj0OmZSXFKAs1e6YsbkRM+FaPtwcA9Fek4EIp8+Wcq1HeMdR+uCDzguYjjHGBBkO4IMljqdtYLsj7PVQKiI8tmlnHbrBkEXAVqLjKRxzfiYFfKk7XA8N5TN8oOGnw9o77ic/bdeUoJmzgcCP/dfB+1UDCXKXMcA1vU89kzf7XDiHAZgNO0nJmSVCl3fSZKQA/F1/B90LEGVDofnpOJMLCpLFN8WHAGS5o4xldIafMgDSLW8rhLHOJhXPjUORQ1oiZDrTxQ+3FKdlNvPKzZsaac7r454z8g2/9ourhqV1PIJdS1hrM/g6ByTOxrPhn1+e6Yv0vENEDhRZ/ZGmjn+TQTSuyyuu0nKG9ky1jC7/aoAWNQuN7jRXrFOH8HAUhArTdNPTYEHBqprAdHh5ccLUpv4yRvCNqxmmTu+c/95vVDQ3s2RndGqwp9xSNh+7nSBfrW+78f7fY4X0X/7wBQ/1sp7GbxWuWVkvkgDC5pvaUKuiddoQoYJysXUOADMBh6FBO/zZh76HSE="
-        # Undocumented flag to create a draft release
-        draft: true
-        file_glob: true
-        file: FreeOrionSDK_*.tar.bz2
-        skip_cleanup: true
-        on:
-          tags: true
+
+    - name: Build FreeOrionSDK for Windows (x86)
+      <<: *_build_sdk_common
+      os: windows
+      env:
+        - RELEASES_FILE=FreeOrionSDK_*.zip
+        - RELEASES_TAG_NAME=$TRAVIS_TAG
+      script:
+        - cmake -G "Visual Studio 15" -T v141 -A Win32 ..
```

Comment:
Should we drop XP support? @geoffthemedio

Reply:
You are trying to hard.  XP EOL is public knowledge.  Also you don't seem to read the  build logs of the stuff you build, do you?  Otherwise you would have seen the deprecation messages regarding the toolkit support (if you don't known what's going on look up MSB8051).  Which is especially strange as you're advocating v142.

TL;DR: You're not in charge of deciding for or against XP support, Microsoft is.

Reply:
Keeping older versions supported is fine as long as it doesn't impede work severely, but if it does, I think XP is old enough now that we can move on. There are likely some people still using it, but it is almost 20 years old now...